Specs/DateTime/Arabic/DateExtractor.json

```diff
@@ -27,72 +25,66 @@
   },
   {
     "Input": "سأعود في 1 يناير",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "java, javascript, python",
     "Results": [
       {
-        "Text": "1  يناير ",
-        "Start": -1,
-        "Length": 9,
+        "Text": "1 يناير",
```

Comment:
Why do some Text seem reversed compared to the Input and some don't?

Reply:
It is because Arabic uses right to left writing, and so it is better to look at the specs with the same text direction to avoid these artifacts. 
In left to right direction, text can show a different order (especially when mixed with numbers or latin characters), but in right to left direction the order is the same as in the input.
Note that the same also happens in the Number specs, for example from the NumberRangeModel:
"Input": "نطاق العدد هو 20 إلى 100"
"Text": "20 إلى 100"

Reply:
I wonder if we should improve the test code to make this easier to understand. Let's not make this change now.
Thanks.